pkg/serving/config_changes_test.go

```diff
@@ -139,6 +139,43 @@ func testUpdateEnvVarsModify(t *testing.T, revision *servingv1alpha1.RevisionTem
 	}
 }
 
+func TestUpdateContainerPort(t *testing.T) {
+	template, _ := getV1alpha1Config()
+	err := UpdateContainerPort(template, 8080)
```

Comment:
Why do you use the default port for the update ? I think for clarities sake its better to use something different.

Reply:
Modified